### ISSUE TITLE
runTizenIDE.sh contains rm -rf ~/.local

### ISSUE THREAD

**Issue by chwar** [kamildzi/Tizen4Docker#25]
Hi,

one of the scripts is hugely dangerous: `runTizenIDE.sh` contains the following lines:

```
rm -rf ~/.local
ln -s $VOLUME_DATA_DIR ~/.local
``` 
Had I not gotten an error before its execution (same as https://github.com/kamildzi/Tizen4Docker/issues/12), my entire ~/.local folder would have been deleted! This is **extremely dangerous** as ~/.local contains almost all of my apps (in my case this is over 20 Gb of data).

Besides, commenting these lines and executing the script still resulted in a successful installation of the docker image. The IDE works as expected.

**Comment by kamildzi** [comment]
Hello, 

Thank you for keeping an eye over the code! 

Are you sure your data were in danger? I did a quick review and to be honest, I really doubt that. :smile: 

Please note following: 
- `runTizenIDE.sh` runs **inside docker container**, not on your host environment, 
- therefore your host path `~/.local` is not equal to `~/.local` inside the container, 
- `~/.local` inside the container actually points to data in the docker volume: `$LOCAL_TIZEN_STUDIO_DATA_DIRECTORY/_EXT_TIZEN4DOCKER_/.local` (please note your `.env` configuration),
- this directory is used to store tizen environment local settings (especially gnome keyrings).

You might be interested to see more code explanations. Please forgive me my ugly drawings., hope it will be useful. :rofl: 
![obraz](https://user-images.githubusercontent.com/12057520/103162872-91e28300-47f6-11eb-8b86-78dbe719a260.png)

**Comment by chwar** [comment]
Oh my bad, then. I didn't think about the fact that it's run directly within the container. 
I panicked when seeing that, especially more since I experienced loss of ownership of my home folder with the script (not sure what caused it, I had the same issue described in https://github.com/kamildzi/Tizen4Docker/issues/12 and I think I gave the wrong uid to the script (1000 instead of 1001)...).
Thank you for your explanation, this can be closed now :)
